### ISSUE TITLE
Understanding readme (negation)

### ISSUE THREAD

**Issue by epogrebnyak** [microsoft/EconML#620]
> (i.e. there is no unobserved variable not included in X, W that simultaneously has an effect on both T and Y) 

All unobserved variables that have an effect on both T and Y are included in X, W, is that the meaning?

**Comment by kbattocchi** [comment]
Yes, that's basically right, except that terminology-wise if it's in X or W it must have been observed (since it's recorded in the data set).  In other words, the unobserved things by definition are not in X or W, and it's okay there are some such unobserved things that affect only T and/or some of them that affect only Y, but the method is not guaranteed to produce unbiased results if there are unobserved things that simultaneously affect both T and Y.
